intake/catalog/base.py

```diff
@@ -721,12 +722,12 @@ def search(self, *args, **kwargs):
                    'source_id': self._source_id}
         response = requests.post(
             url=self.source_url, **self._get_http_args({}),
-            data=msgpack.packb(request, use_bin_type=True))
+            data=msgpack.packb(request, default=encode_datetime, use_bin_type=True))
```

Comment:
I wonder if it is worth adding `pack_kwargs`? Since we use the same parameters every time, might as well make it easy to keep all the calls in sync.